<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1111</width>
    <height>637</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout_2">
    <item>
     <layout class="QVBoxLayout" name="layout_vert_menu">
      <item>
       <widget class="QGraphicsView" name="grph_capture_view">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>240</height>
         </size>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="grp_device_settings">
        <property name="title">
         <string>Device Settings</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <property name="spacing">
          <number>9</number>
         </property>
         <item row="0" column="0">
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Image Preprocessing</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QCheckBox" name="checkBox">
           <property name="text">
            <string>Use Fast Extract</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>Inferencing</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QCheckBox" name="checkBox_2">
           <property name="text">
            <string>Use CPU</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>NSC 2</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QCheckBox" name="checkBox_3">
           <property name="text">
            <string>Use NCS 2</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Device List</string>
           </property>
          </widget>
         </item>
         <item row="4" column="0" colspan="2">
          <widget class="QListView" name="listView">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>90</height>
            </size>
           </property>
          </widget>
         </item>
         <item row="5" column="0" colspan="2">
          <widget class="QPushButton" name="btn_initialization">
           <property name="text">
            <string>Initialize</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_fit_crop">
        <property name="text">
         <string>Fit Crop</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_load_img">
        <property name="text">
         <string>Load Image</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btn_crop_img">
        <property name="text">
         <string>Crop Image</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="layout_vert_detect">
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QLabel" name="lbl_process_time">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Process Time :</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="lbl_result_time">
          <property name="font">
           <font>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>ms</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QGraphicsView" name="grph_detect_result"/>
      </item>
      <item>
       <layout class="QHBoxLayout" name="layout_hor_button_detect">
        <item>
         <widget class="QPushButton" name="btn_enhancement">
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>Image Enhancement</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_detect_obj">
          <property name="text">
           <string>Draw Contour</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_blob_detection">
          <property name="text">
           <string>Blob Detection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_edgeDetection">
          <property name="text">
           <string>Edge Detection</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btn_Laplacian">
          <property name="text">
           <string>Laplacian</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1111</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuImage_Capture">
    <property name="title">
     <string>Image Capture</string>
    </property>
    <addaction name="actionOpen_Camera"/>
   </widget>
   <widget class="QMenu" name="menuCocnfig">
    <property name="title">
     <string>Config</string>
    </property>
    <addaction name="actionLoad_Model"/>
    <addaction name="actionLoad_Class"/>
   </widget>
   <addaction name="menuMenu"/>
   <addaction name="menuImage_Capture"/>
   <addaction name="menuCocnfig"/>
  </widget>
  <action name="actionLoad_Model">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Load Model</string>
   </property>
  </action>
  <action name="actionLoad_Class">
   <property name="text">
    <string>Load Class</string>
   </property>
  </action>
  <action name="actionOpen_Camera">
   <property name="text">
    <string>Open Camera</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
